safe/impact_functions/generic/continuous_hazard_population/impact_function.py

```diff
@@ -108,37 +110,58 @@ def run(self, layers=None):
         self.validate()
         self.prepare(layers)
 
-        # The 3 category
-        high_t = self.parameters['Categorical thresholds'][2]
-        medium_t = self.parameters['Categorical thresholds'][1]
-        low_t = self.parameters['Categorical thresholds'][0]
+        thresholds = self.parameters['Categorical thresholds']
+
+        # Thresholds must contain 3 thresholds
+        if len(thresholds) != 3:
+            raise FunctionParametersError(
+                'The thresholds must consist of 3 values.')
+
+        # Thresholds must monotonically increasing
+        monotonically_increasing_flag = all(
+            x < y for x, y in zip(thresholds, thresholds[1:]))
+        if not monotonically_increasing_flag:
+            raise FunctionParametersError(
+                'The thresholds must be monotonically increasing')
```

Comment:
Can we make this more layman friendly? How about: "Each threshold should be larger than the previous"